src/process.rs

```diff
@@ -1,11 +1,10 @@
 //! Process management.
 
 #[cfg(unix)]
```

Comment:
Given that socket-based activation is a _systemd_ thing and therefore a Linux thing, perhaps we should only make it available on Linux?